### PR TITLE
An introduction to Expo

### PR DESCRIPTION
**Pull request by Jonak-Adipta-Kalita**
# We can't leave Expo behind too!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,13 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## Expo
+
+- [Expo Crash Course by Traversy Media](https://www.youtube.com/watch?v=Hf4MJH0jDb4)
+- [Expo Snack](https://snack.expo.dev/)
```
Comment (shahednasser) [inline README.md:198]: I don't think this one qualifies so please remove it.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @Jonak-Adipta-Kalita I have a comment below.

File: README.md (hunk 1 above)
Comment (Jonak-Adipta-Kalita) [reply README.md:198]: Ok

**Comment by Jonak-Adipta-Kalita** [comment]
please check if it is okay now

**Comment by Jonak-Adipta-Kalita** [comment]
@shahednasser please check if its okay now

**Review by shahednasser — CHANGES_REQUESTED** [review]
Sorry I forgot to mention this earlier but please add Expo to the table of contents

**Comment by Jonak-Adipta-Kalita** [comment]
> Sorry I forgot to mention this earlier but please add Expo to the table of contents

I dont even know why i forgot to do that. Sorry 😅😅

**Comment by Jonak-Adipta-Kalita** [comment]
Also I am the guy who Introduced to HTML in this repo

**Comment by shahednasser** [comment]
Great thank you!

**Comment by Jonak-Adipta-Kalita** [comment]
> Great thank you!

My Pleasure!!

**Pull request merged**